doc/whats_new.rst

```diff
@@ -7,312 +7,139 @@
 What's new?
 ===========
 
-Here we list a changelog of MNE-BIDS.
+.. currentmodule:: mne_bids
+
+.. _changes_0_5:
+
+Version 0.5 (2020-10-xx)
+------------------------
+
+This is a **big** release with lots of changes, many of them breaking existing
+code. But do not fear: migration is easy, and you will **love** what we have
+been cooking for you!
 
 .. contents:: Contents
    :local:
    :depth: 3
 
-.. currentmodule:: mne_bids
+Notable changes
+~~~~~~~~~~~~~~~
+- We introduce `mne_bids.BIDSPath`, a new class for all BIDS file and folder
+  operations. All functions in MNE-BIDS that previously accepted filenames
+  and folder locations (e.g. ``bids_root``) have been updated to work with
+  ``BIDSPath``. Others have been removed.
+  Consequently, you will need to update your existing code, too.
+  See the `API changes`_ section for an overview of which functions have
+  changed or have been removed, and follow
+  :ref:`this introduction<bidspath-intro>`
+  to learn about the basics of ``BIDSPath``. Don't worry – it's going to be a
+  breeze! 🤗
+
+- MNE-BIDS now requires MNE-Python 0.21.
+
+- The new function :func:`mne_bids.make_report` can help you populate a
+  paragraph of your next paper's methods section!
 
-.. _current:
+- You can now interactively mark channels as bad using
+  :func:`mne_bids.mark_bad_channels`.
+
+- Elekta/Neuromag/MEGIN fine-calibration and crosstalk files can now be stored
+  in your BIDS dataset via :func:`mne_bids.write_meg_calibration` and
+  :func:`mne_bids.write_meg_crosstalk`.
+
+- When writing a raw file that contains annotations, these will now be
+  converted to and stored as events by :func:`mne_bids.write_raw_bids`.
+
+
+Authors
+~~~~~~~
+The following people have contributed to this release of MNE-BIDS:
 
-Current
--------
+- `Adam Li`_
+- `Alexandre Gramfort`_
+- `Alex Rockhill`_
+- `Richard Höchenberger`_ 
+- `Stefan Appelhoff`_
```

Comment:
need to add Mainak, maybe you forgot him because he didn't make a commit for this release?

Reply:
Right, I only included those who committed, so I forgot about Mainak's extensive code reviews! Will add.

Reply:
Resolved in ef4da08